.github/workflows/codeql.yml

```diff
@@ -0,0 +1,42 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches:
+      - master
+      - teleport
```

Comment:
does the branch name is not teleport.1 ?

Reply:
I haven't ported to `teleport` yet, because there is a commit there (it is being backported in this PR). Ideally, we would have only `teleport` (branch with our changes) and `master/main` with Microsoft fork.